src/stratis_cli/_errors.py

```diff
@@ -33,6 +33,32 @@ class StratisCliRuntimeError(StratisCliError):
     """
 
 
+class StratisCliPropertyNotFoundError(StratisCliRuntimeError):
+    """
+    Exception raised when a requested property from FetchProperties DBus interface
+    does not exist.
+    """
+
+    def __init__(self, type_name, prop_name):
+        """ Initializer.
+
+            :param str type_name: the name of the type that does not support this property name
+            :param str prop_name: the property that did not exist
+        """
+        # pylint: disable=super-init-not-called
+        self.type_name = type_name
+        self.prop_name = prop_name
+
+    # pylint: disable=fixme
+    # FIXME: Find a way to test the string conversion method. This will
+    # require a call directly to the DBus interface.
```

Comment:
Instead of putting ```no cover``` pragma here, consider putting it on the whole class. See ```StratisCliUnkownInterfaceError``` for an example. I think that the reasoning would be kind of the same.

Reply:
Resolved.